analyses/mutational-signatures/scripts/extract_signatures_benchmark.R

```diff
@@ -0,0 +1,126 @@
+# Written by Stephanie Spielman and adapted from code by Anna R. Poetsch, Candace L. Savonen, and J. Taroni
+# Determine optimal k for de novo mutatational signature analysis as well as robustness to random seed.
+#   Analysis is performed with a small number of iterations (1000) and tests k 2:8 (previous small runs suggest this range is appropraite) provided a given model (multinomial or poisson) and random seed.
+#   Needs up to 20 GB RAM but is single-threaded
```

Comment:
We might want to add this to the README for the module, but I think that can wait as your initial comment suggests.

Reply:
Sounds good - will update README in next PR with finalized analysis details.